apps/velo-external-db/test/e2e/app_data.e2e.spec.ts

```diff
@@ -49,106 +50,95 @@ describe(`Velo External DB Data REST API: ${currentDbImplementationName()}`,  ()
-
-        await expect(streamToArray(response.data)).resolves.toEqual(expectedItems)
-        await expect(data.queryCollectionAsArray(ctx.collectionName, [], undefined, authOwner)).resolves.toEqual(expect.toIncludeSameMembers(
-            [ 
-                ...expectedItems, 
-                data.pagingMetadata(ctx.items.length, ctx.items.length)
-            ])
-        )
+        expect(response.data).toEqual({ results: expect.toIncludeSameMembers(ctx.items) })
+    
+        await expect(data.queryCollectionAsArray(ctx.collectionName, [], undefined, authOwner)).resolves.toEqual({
+            items: expect.toIncludeSameMembers(ctx.items),
+            pagingMetadata: data.pagingMetadata(ctx.items.length, ctx.items.length)
+        })
     })
 
     testIfSupportedOperationsIncludes(supportedOperations, [ AtomicBulkInsert, PrimaryKey ])('insert api should fail if item already exists', async() => {
         await schema.givenCollection(ctx.collectionName, [ctx.column], authOwner)
         await data.givenItems([ ctx.items[1] ], ctx.collectionName, authAdmin)
 
-        const response = axiosInstance.post('/data/insert', data.insertRequest(ctx.collectionName, ctx.items, false),  { responseType: 'stream', ...authAdmin })
+        const response = axiosInstance.post('/data/insert', data.insertRequest(ctx.collectionName, ctx.items), { ...authAdmin })
 
-        const expectedItems = [dataSpi.QueryResponsePart.item(ctx.items[1])]
+        const expectedItems = [ctx.items[1]]
 
         await expect(response).rejects.toThrow('409')
 
-        await expect(data.queryCollectionAsArray(ctx.collectionName, [], undefined, authOwner)).resolves.toEqual(expect.toIncludeAllMembers(
-            [ 
-                ...expectedItems, 
-                data.pagingMetadata(expectedItems.length, expectedItems.length)
-            ])
-        )
+        await expect(data.queryCollectionAsArray(ctx.collectionName, [], undefined, authOwner)).resolves.toEqual({
+            items: expectedItems,
+            pagingMetadata: data.pagingMetadata(expectedItems.length, expectedItems.length)
+        })
+
+
     })
 
     testIfSupportedOperationsIncludes(supportedOperations, [NonAtomicBulkInsert, PrimaryKey])('insert api should throw 409 error if item already exists and continue inserting the rest', async() => {
         await schema.givenCollection(ctx.collectionName, [ctx.column], authOwner)
         await data.givenItems([ ctx.items[1] ], ctx.collectionName, authAdmin)
 
-        const response = axiosInstance.post('/data/insert', data.insertRequest(ctx.collectionName, ctx.items, false),  { responseType: 'stream', ...authAdmin })
-
-        const expectedItems = ctx.items.map(i => dataSpi.QueryResponsePart.item(i))
+        const response = axiosInstance.post('/data/insert', data.insertRequest(ctx.collectionName, ctx.items),  { ...authAdmin })
                 
         await expect(response).rejects.toThrow('409')
```

Comment:
I think according to the new SPI, we should get a response with an array - item of each insert request - all of the items should contain the item execpt the one with the error right? I don't see it in the tests.